tests/service/appService.test.ts

```diff
@@ -1,16 +1,15 @@
+import nzbFacade from 'src/facade/nzbFacade';
```

Comment:
Jest is now configured to support the root path, so I updated all of them for consistency.